blob/service.go

```diff
@@ -300,6 +296,8 @@ func (s *Service) getAll(
 // To ensure that blob was included in a specific height, we need:
 // 1. verify the provided commitment by recomputing it;
 // 2. verify the provided Proof against subtree roots that were used in 1.;
+// Note: this method can be deprecated because it's doing processing that can
+// be done locally.
```

Comment:
Since our API operates with header's height, deprecating this method requires users to download the header to get the hash.
So, I agree that we can deprecate this method in favour of proof public `validate` methods but only after migration from heights to hashes.

Reply:
cool, we can create an issue for it

Reply:
https://github.com/celestiaorg/celestia-node/issues/3919

Reply:
I think we had somewhere such an issue or task inside the EPIC. I will double-check and close the issue in this case